flare_emu.py

```diff
@@ -463,10 +463,7 @@ def getEmuWideString(self, addr):
 
     # returns a <size> string of bytes read from <addr>
     def getEmuBytes(self, addr, size):
-        out = ""
-        for i in range(addr, addr+size):
-            out += self.uc.mem_read(i, 1)
-        return out
+        return self.uc.mem_read(addr, size)

```

Comment:
Nice addition, but can you please return a **str** type instead of a **bytearray**? This would conform with IDA Pro's get_bytes/GetManyBytes API. Thanks!

Reply:
Changed as requested :)

Reply:
Thank you!